src/pages/learn/introspection.mdx

```diff
@@ -146,25 +144,36 @@ tell us what these are lists of.
 }
 ```
 
-Let's end with a feature of the introspection system particularly useful
-for tooling; let's ask the system for documentation!
+Let's end with a feature of the introspection system particularly useful for tooling; let's ask the system for documentation:
 
 ```graphql
 # { "graphiql": true }
-{
+query {
   __type(name: "Droid") {
     name
     description
   }
 }
 ```
 
-So we can access the documentation about the type system using introspection,
-and create documentation browsers, or rich IDE experiences.
+As demonstrated above, we can access the documentation about the type system using introspection and create documentation browsers or rich IDE experiences.
+
+This has just scratched the surface of the introspection system; we can query for Enum type values, what Interface types another type implements, and more. We can even introspect on the introspection system itself.
+
+To see an example of a specification-compliant GraphQL query introspection system implemented in code, you can view the [introspection.ts](https://github.com/graphql/graphql-js/blob/main/src/type/introspection.ts) file in GraphQL.js repository.
```

Comment:
I've switched this to a permalink, though I'm not entirely sure I should... Trade-offs either way I think.

I also think we should refer to "GraphQL.js" as "the reference implementation" as it makes it clearer why we're directing people there.

```suggestion
To see an example of a specification-compliant GraphQL query introspection system implemented in code, you can view [src/type/introspection.ts](https://github.com/graphql/graphql-js/blob/e9b6b626f6f6aa379bb8f8c48df40d0c02a26082/src/type/introspection.ts) in the reference implementation.
```